main.go

```diff
@@ -132,7 +147,12 @@ func mainE() error {
 			`%cn`: "GIT_CLONE_COMMIT_COMMITER_NAME",
 			`%ce`: "GIT_CLONE_COMMIT_COMMITER_EMAIL",
 		} {
-			if err := printLogAndExportEnv(gitCmd, format, env); err != nil {
+			if env == "GIT_CLONE_COMMIT_MESSAGE_BODY" {
```

Comment:
In Go you can evaluate the bool expression in the function's method:
Instead of:
```
if env == "GIT_CLONE_COMMIT_MESSAGE_BODY" {
	if err := printLogAndExportEnv(gitCmd, format, env, true); err != nil {
		return fmt.Errorf("gitCmd log failed, error: %v", err)
	}
}

if err := printLogAndExportEnv(gitCmd, format, env, false); err != nil {
	return fmt.Errorf("gitCmd log failed, error: %v", err)
}
```

Use:
```

if err := printLogAndExportEnv(gitCmd, format, env,(env == "GIT_CLONE_COMMIT_MESSAGE_BODY")); err != nil {
	return fmt.Errorf("gitCmd log failed, error: %v", err)
}
```

Reply:
Fixed.